pepper-apis/src/main/java/org/broadinstitute/ddp/util/LiquibaseUtil.java

```diff
@@ -104,18 +106,18 @@ private void runPepperAPIsGlobalMigrations() throws LiquibaseException {
      *
      * @throws LiquibaseException if something goes wrong
      */
-    private void runHousekeepingGlobalMigrations() throws LiquibaseException {
+    private void runHousekeepingGlobalMigrations() throws LiquibaseException, SQLException {
         runMigrations(HOUSEKEEPING_GLOBAL_MIGRATIONS);
     }
 
     /**
      * Runs the specific changelog file. When migration fails, this will attempt to rollback the changes, as applicable.
      */
-    private void runMigrations(String changelogFile) throws LiquibaseException {
+    private void runMigrations(String changelogFile) throws LiquibaseException, SQLException {
         Liquibase liquibase = null;
         String tag = null;
         try {
-            liquibase = new Liquibase(changelogFile, new FileSystemResourceAccessor(), new JdbcConnection(conn));
+            liquibase = new Liquibase(changelogFile, new ClassLoaderResourceAccessor(), new JdbcConnection(dataSource.getConnection()));
             logLocks(liquibase.listLocks());
```

Comment:
Drop FileSystemResourceAccessor and use ClassLoaderResourceAccessor: the key to read config files from jar file.